server/controllers/payment.js

```diff
@@ -0,0 +1,53 @@
+const Payment = require("../models/Payment");
+const stripe = require("../utils/stripe");
+const asyncHandler = require("express-async-handler");
+const stripe = require("stripe")(
+  "sk_test_51KcFwoAZ9gJcnOgDUNatBvPLDR4J7tfHodr55G24tP3UZgO3nyWtvE272SaqYqNHtpf22UTCUX2hDM9s0XXW4W5X0064ASBu8x"
+);
```

Comment:
We don't actually want to expose our test key in the back-end, this is better to be placed in the environment variables. Please address this ASAP